### DISCUSSION TITLE
Please review my github action for hackathon :)

### DISCUSSION THREAD

**Discussion by TheShubham99** [EddieHubCommunity/support#208] [Show and tell]
## What it does?
Sends out a personalized email appreciating the contribution with some Badges and Image Processed Personalized Material.

link - https://github.com/TheShubham99/YouRock

**Comment by eddiejaoude** [comment]
It looks great, screenshots below 👇 .

I wonder if replying to the Issue would be better than email, because most people will have their email set to private?

<img width="1466" alt="Screenshot 2020-09-18 at 09 01 17" src="https://user-images.githubusercontent.com/624760/93572483-d1930680-f98d-11ea-8d40-71b68247c722.png">
<img width="1458" alt="Screenshot 2020-09-18 at 09 01 25" src="https://user-images.githubusercontent.com/624760/93572465-ccce5280-f98d-11ea-9157-af0263faac8e.png">

*note: I think there are some spacing issues in the text*

**Comment by TheShubham99** [comment]
Hey Eddie,

First of all, thank you for testing it.

Yes. I am thinking of handling the exception by pasting it in the comments.

Like,

If the email is private --> Then exception is raised --> handling the
exception --> Paste the result into the comments.

But how to do it? 🤔
How to do exception handling in Github actions?

Regards,
Prathamesh / TheShubham99

On Fri, 18 Sep 2020 at 13:34, Eddie Jaoude <notifications@github.com> wrote:

> It looks great, screenshots below 👇 .
>
> I wonder if replying to the Issue would be better than email, because most
> people will have their email set to private?
>
> [image: Screenshot 2020-09-18 at 09 01 17]
> <https://user-images.githubusercontent.com/624760/93572483-d1930680-f98d-11ea-8d40-71b68247c722.png>
>
> [image: Screenshot 2020-09-18 at 09 01 25]
> <https://user-images.githubusercontent.com/624760/93572465-ccce5280-f98d-11ea-9157-af0263faac8e.png>
>
> *note: I think there are some spacing issues in the text*
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/EddieJaoudeCommunity/support/discussions/208#discussioncomment-80923>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AE4WMGBB3I5H2W6UMAIPRV3SGMIA7ANCNFSM4RQNPBBA>
> .
>


-- 

PRATHAMESH SAHASRABHOJANE
Full Stack Developer | Mixed Reality Enthusiast
p: 07588826076
other email: shubham.1971999@gmail.com
http://prathamesh.me/Work
[image: twitter icon] <https://twitter.com/pratham_sh99> [image: linkedin
icon] <https://www.linkedin.com/in/prathamesh-sahasrabhojane-4b3b12148/>
[image:
instagram icon] <https://www.instagram.com/pratham_99>

**Reply by eddiejaoude** [reply]
`try/catch`

**Comment by TheShubham99** [comment]
In YAML? 🤔

On Fri, 18 Sep 2020 at 13:55, Eddie Jaoude <notifications@github.com> wrote:

> try/catch
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/EddieJaoudeCommunity/support/discussions/208#discussioncomment-80937>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AE4WMGHLQPODYZVDXEGVBSTSGMKQ3ANCNFSM4RQNPBBA>
> .
>


-- 

PRATHAMESH SAHASRABHOJANE
Full Stack Developer | Mixed Reality Enthusiast
p: 07588826076
other email: shubham.1971999@gmail.com
http://prathamesh.me/Work
[image: twitter icon] <https://twitter.com/pratham_sh99> [image: linkedin
icon] <https://www.linkedin.com/in/prathamesh-sahasrabhojane-4b3b12148/>
[image:
instagram icon] <https://www.instagram.com/pratham_99>

**Reply by eddiejaoude** [reply]
no, in the code
